hal/tests/TESTS/mbed_hal/stack_size_unification/CMakeLists.txt

```diff
@@ -3,9 +3,40 @@
 
 include(mbed_greentea)
 
+if("TARGET_CORTEX_A" IN_LIST MBED_TARGET_DEFINITIONS)
+    set(TEST_SKIPPED "Cortex-A target not supported for this test")
+endif()
+
+macro(GetMemSize resultVar memRegion)
+  # ARGN holds all arguments to function after last named one
+  foreach(ITR ${ARGN})
```

Comment:
I really dislike the way CMake does this variadic argument handling. Could we just create a named argument for the `MBED_CONFIG_DEFINITIONS` list? Or just use it directly as it's global anyway?